Modularization (#458)
dailymotion/vast-client-js: README.md

In addition to fetching and parsing a VAST resource, **VASTClient** provides options to filter a sequence of calls based on count and time of execution, together with the possibility to track URLs using **VASTTracker**.

> If you need to directly parse a VAST XML and  also follow any wrappers chain, you can use the  `parseVAST` method from the  **VASTClient** :

Reviewer: ```suggestion
If you need to directly parse a VAST XML and also follow any wrappers chain, you can use the  `parseVAST` method from the  **VASTClient** :
```

#### Browser script

A pre-bundled version of VAST Client JS is available: [`vast-client.min.js`](dist/vast-client.min.js) [minified].

> If you want to use this file, download it place it your project at your prefered location.
> Then, make sure that your main file is in type module as below.

Reviewer: ```suggestion
To use it, either host it on your CDN or locally in your project. If you're using a script tag make sure to set the type property to module like below 
```

> 
> ```html

Reviewer: ```suggestion
_your index.html_
```html
```

<script type="module" src="your-main-file.js"></script>
```

> 
> ```javascript

Reviewer: ```suggestion
_main.js_
```javascript
```
